components/DailyCrusher/DailyCrusher.js

```diff
@@ -0,0 +1,171 @@
+  const [isCrusherDetailsVisible, setIsCrusherDetailsVisible] = useState(false);
+
+  function toggleVisibility() {
+    setIsCrusherDetailsVisible(!isCrusherDetailsVisible);
+  }
+
+  function getCurrentDayWorkout() {
+    const today = new Date().getDay();
+    const workoutSchedule = {
+      1: workouts[5],
+      2: workouts[1],
+      3: workouts[2],
+      4: workouts[3],
+      5: workouts[4],
+      6: workouts[5],
+      0: workouts[0],
+    };
+
+    return workoutSchedule[today] || null;
+  }
+
+  const currentWorkout = getCurrentDayWorkout();
+
+  if (!currentWorkout) {
+    return <p>No workout scheduled for today. Rest day!</p>;
+  }
+
+  const muscleGroups = [];
+
+  const enrichedExercises = currentWorkout.exercises.map((exercise) => {
+    const foundExercise = findExerciseById(exercises, exercise.exerciseId);
+    if (foundExercise) {
+      muscleGroups.push(...foundExercise.muscleGroups);
+      return { ...exercise, name: foundExercise.name };
+    }
+    return exercise;
+  });
+
+  const uniqueMuscleGroups = [...new Set(muscleGroups)];
+
+  const dailyCrusher = {
+    ...currentWorkout,
+    exercises: enrichedExercises,
+    muscleGroups: uniqueMuscleGroups,
+  };
+
+  return (
+    <StyledCrusherWrapper>
+      <StyledHeadline>Crusher of the Day 💪</StyledHeadline>
+      <StyledArticleWrap>
+        <StyledDescriptionText>
+          Get in shape with our crusher workouts! Every day we suggest a
+          different workout for you to try out.
+        </StyledDescriptionText>
+
+        <StyledLink href="/workoutlist">
+          <Workout workout={dailyCrusher} spotlightMode={true} />
+        </StyledLink>
+        <StyledPostionRelative>
+          {isCrusherDetailsVisible ? null : (
+            <StyledAbsoluteToggleButton onClick={toggleVisibility}>
+              Show More
+            </StyledAbsoluteToggleButton>
+          )}
+          <StyledPreviewFilter
+            $showMore={
+              isCrusherDetailsVisible ? "filter: none; color: black;" : null
+            }
+          >
+            <h3>{dailyCrusher.title}</h3>
+            <StyledDescriptionText>
+              {dailyCrusher.description1}
+            </StyledDescriptionText>
+          </StyledPreviewFilter>
+        </StyledPostionRelative>
+        {isCrusherDetailsVisible ? (
+          <>
+            {dailyCrusher.exercises.map((exercise) => {
+              return (
+                <StyledExerciseSetsReps key={uid()}>
+                  <StyledBoldText>{exercise.name}: </StyledBoldText> Sets:{" "}
+                  {exercise.sets} / Reps: {exercise.reps}
+                </StyledExerciseSetsReps>
+              );
+            })}
+            <StyledDescriptionText>
+              {dailyCrusher.description2}
+            </StyledDescriptionText>
+            <StyledDescriptionText>
+              {dailyCrusher.description3}
+            </StyledDescriptionText>
+          </>
+        ) : null}
```

Comment:
```suggestion
        {isCrusherDetailsVisible && (
          <>
            {dailyCrusher.exercises.map((exercise) => {
              return (
                <StyledExerciseSetsReps key={uid()}>
                  <StyledBoldText>{exercise.name}: </StyledBoldText> Sets:{" "}
                  {exercise.sets} / Reps: {exercise.reps}
                </StyledExerciseSetsReps>
              );
            })}
            <StyledDescriptionText>
              {dailyCrusher.description2}
            </StyledDescriptionText>
            <StyledDescriptionText>
              {dailyCrusher.description3}
            </StyledDescriptionText>
          </>
        )}
```
If you only have null after the : in the ternary operator, you can also use the shorter notation with &&. I think that's clearer, cleaner notation.

Reply:
We as a group decided that we want to use the ternary operators for all conditional renderings :)